<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="LAPTOP-U8CM5F9C" name="test_Ng_designing_test_class.TestNG_Keywords" tests="9" failures="0" timestamp="2021-03-29T10:57:39 IST" time="8.677" errors="1">
  <testcase name="verify_ProfNo1" time="0.004" classname="test_Ng_designing_test_class.TestNG_Keywords"/>
  <system-out>
--verify profile no 1---  </system-out>
  <testcase name="verify_ProfNo1" time="0.008" classname="test_Ng_designing_test_class.TestNG_Keywords"/>
  <system-out>
--verify profile no 1---  </system-out>
  <testcase name="verify_ProfNo1" time="0.009" classname="test_Ng_designing_test_class.TestNG_Keywords"/>
  <system-out>
--verify profile no 1---  </system-out>
  <testcase name="verify_ProfNo1" time="0.005" classname="test_Ng_designing_test_class.TestNG_Keywords"/>
  <system-out>
--verify profile no 1---  </system-out>
  <testcase name="verify_ProfNo1" time="0.007" classname="test_Ng_designing_test_class.TestNG_Keywords"/>
  <system-out>
--verify profile no 1---  </system-out>
  <testcase name="_clic_on_serchbox" time="0.010" classname="test_Ng_designing_test_class.TestNG_Keywords"/>
  <system-out>
---user shoud be able to serch prod----  </system-out>
  <testcase name="click_on_Cart" time="8.627" classname="test_Ng_designing_test_class.TestNG_Keywords">
    <error type="org.testng.internal.thread.ThreadTimeoutException" message="Method test_Ng_designing_test_class.TestNG_Keywords.click_on_Cart() didn&#039;t finish within the time-out 8000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method test_Ng_designing_test_class.TestNG_Keywords.click_on_Cart() didn't finish within the time-out 8000
at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNewExecutor(MethodInvocationHelper.java:371)
at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:282)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:605)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- click_on_Cart -->
  <system-out/>
  <testcase name="verify_ProfNo2" time="0.007" classname="test_Ng_designing_test_class.TestNG_Keywords"/>
  <system-out>
---verify profile no 2----  </system-out>
  <testcase name="clicOnlogo" classname="test_Ng_designing_test_class.TestNG_Keywords">
    <skipped/>
  </testcase> <!-- clicOnlogo -->
  <system-out/>
</testsuite> <!-- test_Ng_designing_test_class.TestNG_Keywords -->
